server/app/test_bids_payments.py

```diff
@@ -0,0 +1,175 @@
+# This is a draft test before ERM review finished;
+# Test CRUD on table Bids and Payments;
+# Assume user and task are already created with no problems before testing Bids and Payments.
+from django.test import TestCase
+from django.core.exceptions import ValidationError
+from .models import Users, Tasks, Bids, Payments
+from datetime import datetime, timedelta
+
+class BidsPaymentsTestCase(TestCase):
+
+    def setUp(self):
+        # Create a test user
+        self.user = Users.objects.create(
+            email="testuser@example.com",
+            mobile="1234567890",
+            password_hash="hashed_password",
+            status="active"
+        )
+
+        # Create a test task
+        self.task = Tasks.objects.create(
+            owner_id=self.user,
+            title="Test Task",
+            description="Task Description",
+            location="Test Location",
+            deadline=datetime.now() + timedelta(days=5),
+            status="open"
+        )
+
+    def test_create_bid(self):
+        # Create a bid
+        bid = Bids.objects.create(
+            task_id=self.task,
+            bidder_id=self.user,
+            price="100",
```

Comment:
I think it would make more sense for the price field to be stored as a number rather than a string.

Reply:
@0XFF-96 @yunho7687 @2XUID @K1rk3y  I am not sure... Price and estimated_price are designed as varchar according to ERD from Jimmy?  https://cottony-rover-487.notion.site/ERD-4838acae80e34c2ab001e3446956d178. From model code they are designed as charfield?    Jimmy pls help discuss what they will be.

Reply:
Yes, all estimated_price are deliberately designed as string value for better representation.

https://stackoverflow.com/questions/35709595/why-would-you-use-a-string-in-json-to-represent-a-decimal-number

Reply:
Fair enough, I was not aware that the field was deliberately chosen to be a string